From a36ee7501f68ad7ebcfe31f9659430b9d2c3ddd1 Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Thu, 11 Jun 2015 16:19:49 +0900
Subject: [PATCH 1/1] mpi: Support FreeBSD 10 or later.

* mpi/config.links: Include FreeBSD 10 to 29.

--

Thanks to Yuta SATOH.

GnuPG-bug-id: 1936, 1974
---
 mpi/config.links | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/mpi/config.links b/mpi/config.links
index 2fb5e8a..3ead4f0 100644
--- a/mpi/config.links
+++ b/mpi/config.links
@@ -50,11 +50,12 @@ case "${host}" in
        path=""
        mpi_cpu_arch="x86"
        ;;
-    i[3467]86*-*-openbsd*      | \
-    i[3467]86*-*-freebsd*-elf  | \
-    i[3467]86*-*-freebsd[3-9]* | \
-    i[3467]86*-*-freebsdelf*   | \
-    i[3467]86*-*-netbsd*       | \
+    i[3467]86*-*-openbsd*         | \
+    i[3467]86*-*-freebsd*-elf     | \
+    i[3467]86*-*-freebsd[3-9]*    | \
+    i[3467]86*-*-freebsd[12][0-9]*| \
+    i[3467]86*-*-freebsdelf*      | \
+    i[3467]86*-*-netbsd*          | \
     i[3467]86*-*-k*bsd*)
        echo '#define ELF_SYNTAX' >>./mpi/asm-syntax.h
        cat  $srcdir/mpi/i386/syntax.h	   >>./mpi/asm-syntax.h
@@ -64,6 +65,7 @@ case "${host}" in
     i586*-*-openbsd*         | \
     i586*-*-freebsd*-elf     | \
     i586*-*-freebsd[3-9]*    | \
+    i586*-*-freebsd[12][0-9]*| \
     i586*-*-freebsdelf*      | \
     i586*-*-netbsd*	     | \
     i586*-*-k*bsd*	     | \
-- 
2.1.4

